CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# How to contribute to Azure TRE
+
+First of all thank you. Azure TRE is a collective effort to make it easier to create and manage trusted research environment.
+
+## **Did you find a bug?**
+
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/microsoft/AzureTRE/issues).
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/microsoft/AzureTRE/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D). Be sure to include a **title and clear description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+## **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+* Before submitting, please double check on the current coding conventions used.
+
+## **Do you intend to add a new feature or change an existing one?**
+
+* Open a [new feature issue](https://github.com/microsoft/AzureTRE/issues/new?assignees=&labels=feature&template=feature_request.md&title=) so we can first discuss the changes there. After the discussion you may send a PR with relevant changes. Do not send a PR without discussion on an issue.
+
+## **Do you want to contribute to Azure TRE?**
+
+* Awesome, please make sure the steps you offer are independently verified first. Add screenshots when appropriate. File a PR when ready.
```

Comment:
What type of steps are we referring to? Not sure I follow.

Reply:
I updated the description, let me know if it is better now